metadata-analysis/Metadata-Analysis.Rmd

```diff
@@ -28,13 +28,20 @@ library(scales)
 
 visits <-
   read_lines(here("metadata-analysis/metadata/site_visits.txt"))
-
```

Comment:
(This comment will apply to several of the edits in this PR.)

here() should point to your Eva directory, whatever it is named. adding your personal directory to the code will make it fail for everyone else. Run `here::here()` in your console and if it doesn't say [1] "~/sdrive_ce/eva_dz" then let me know so I can help you get your RStudio project set up correctly. If it does say that, then line 30 should work as is. See here for more info: https://here.r-lib.org/
@zabalad

Reply:
Noted, thanks for the tip on using here() ! I removed the commented out file paths that point to my directory.